shell/src/main/java/org/apache/accumulo/shell/commands/ListBulkCommand.java

```diff
@@ -52,22 +47,10 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
 
     List<String> tservers;
 
-    ManagerMonitorInfo stats;
-    ManagerClientService.Iface client = null;
     ClientContext context = shellState.getContext();
-    while (true) {
-      try {
-        client = ManagerClient.getConnectionWithRetry(context);
-        stats = client.getManagerStats(TraceUtil.traceInfo(), context.rpcCreds());
-        break;
-      } catch (ThriftNotActiveServiceException e) {
-        // Let it loop, fetching a new location
-        sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
-      } finally {
-        if (client != null)
-          ManagerClient.close(client, context);
-      }
-    }
+    ManagerMonitorInfo stats = ThriftClientTypes.MANAGER.executeAdminOnManager(context, client -> {
+      return client.getManagerStats(TraceUtil.traceInfo(), context.rpcCreds());
+    });
```

Comment:
Not sure, but this might format better:
```suggestion
    ManagerMonitorInfo stats = ThriftClientTypes.MANAGER.executeAdminOnManager(context,
    client -> client.getManagerStats(TraceUtil.traceInfo(), context.rpcCreds()));
```